### ISSUE TITLE
Can't import replies while using Bulk Import

### ISSUE THREAD

**Issue by Shubhamchinda** [mattermost/mattermost-server#14959]
#### Summary
I'm trying to migrate from Rocket.Chat to Mattermost using [Bulk Import feature](https://docs.mattermost.com/deployment/bulk-loading.html#running-the-bulk-loading-command). I converted all the data in `JSONL` format according to the required [data-format](https://docs.mattermost.com/deployment/bulk-loading-data-format.html#data-format).

## Steps I performed
* To make sure that replies are working fine. I took only one line( message / post object )  from `JSONL` file and ran the bulk loading command. It worked fine.
* Since I was sure that with one JSONL message (post object) I got the reply thread in Mattermost interface. So then I performed the bulk import with all the data ( Including normal messages and messages with replies ) but after completion I didn't the replies at all.

I'm not sure what's the issue here.

**Message object I used in test import**
```
{"type": "post", "post": {"team": "truetrader", "channel": "members-trading-floor", "user": "daddytrader", "message": "Good morning. I'm a newb and have been watching LTRPB for the last 20 minutes. It's on fire. I could have bought in at $68 but I'm restricting myself so paper traded $68 and sold at $98", "props": {}, "create_at": 1587022913894, "replies": [{"user": "beccab007", "message": "nice! i've been practicing short trades on paper... great place to test strategy.", "create_at": 1587022977693}]}}
```

**Sample JSONL format of actual from the after-test import**
```
{"type": "post", "post": {"team": "truetrader", "channel": "members-trading-floor", "user": "daddytrader", "message": "Good morning. I'm a newb and have been watching LTRPB for the last 20 minutes. It's on fire. I could have bought in at $68 but I'm restricting myself so paper traded $68 and sold at $98", "props": {}, "create_at": 1587022913894, "replies": [{"user": "beccab007", "message": "nice! i've been practicing short trades on paper... great place to test strategy.", "create_at": 1587022977693}]}}
{"type": "post", "post": {"team": "truetrader", "channel": "members-trading-floor", "user": "ats194", "message": "Is Dan buying options or the stock when he trades?", "props": {}, "create_at": 1587023841934, "replies": [{"user": "melanie", "message": "stocks", "create_at": 1587023913013}]}}
{"type": "post", "post": {"team": "truetrader", "channel": "members-trading-floor", "user": "sganeshomg", "message": "why $18.4 for urgn?", "props": {}, "create_at": 1587024512814, "replies": [{"user": "fibs", "message": "Just above the weekly pivot of 18.36", "create_at": 1587024628541}]}}
{"type": "post", "post": {"team": "truetrader", "channel": "members-trading-floor", "user": "fibs", "message": "URGN holding till 21.35", "props": {}, "create_at": 1587024556900, "replies": [{"user": "fibs", "message": "Finally paid off, that took a while! Out for 1pt!", "create_at": 1587031716705}]}}
{"type": "post", "post": {"team": "truetrader", "channel": "members-trading-floor", "user": "sganeshomg", "message": "thank u. and how do i find the weekly pivot?", "props": {}, "create_at": 1587024652557, "replies": [{"user": "fibs", "message": "have you gone through the training yet?", "create_at": 1587024704315}]}}
```
**These are only 5 lines, actual data was around 70k lines.**

I'm using 5.22.0 version of MM.

My Os : Ubuntu 18.04.4 LTS

**Comment by amyblais** [comment]
@Shubhamchinda We fixed an issue related to this in v5.22.2 dot release. Would you be open to upgrading to see if your issue gets resolved?

**Comment by Shubhamchinda** [comment]
@amyblais Thank you for your reply. 

Actually I'm [this docker one line command](https://docs.mattermost.com/install/docker-local-machine.html#one-line-docker-install) to install the MM. After running MM I do `docker exec` get inside the container and use bulk import commands there.

Apparently the `docker one line command` for mattermost install the version 5.22.0 . Is there a chance that docker command can  run the 5.22.2release?

**Comment by amyblais** [comment]
Would you be open to trying with v5.23.0? It has the same features as v5.22.0 but with additional bug fixes and stability improvements.

**Comment by Shubhamchinda** [comment]
Yes, Sure.

**Comment by Shubhamchinda** [comment]
@amyblais but as I mentioned I'm using this `docker run --name mattermost-preview -d --publish 8065:8065 --add-host dockerhost:127.0.0.1 mattermost/mattermost-preview` command and it pulls the image of version 5.22.0 .

**Comment by amyblais** [comment]
@cpanato Would you be able to help with this?

**Comment by cpanato** [comment]
you should not use `mattermost-preview` only if you are running for local tests purposes.

I will check this image if is updated and if not will push the latest

**Comment by cpanato** [comment]
ok, the image was not up-to-date. pushed a new update @Shubhamchinda @amyblais

**Comment by Shubhamchinda** [comment]
@cpanato Actually I was only using `mattermost-preview` to create a proof of concept for the data migration from Rocket.Chat to Mattermost. 

Let me know when you update the image.

**Comment by Shubhamchinda** [comment]
> ok, the image was not up-to-date. pushed a new update @Shubhamchinda @amyblais

Great, Let me check `bulk update` now.

**Comment by Shubhamchinda** [comment]
![Screenshot from 2020-07-08 02-01-17](https://user-images.githubusercontent.com/21038543/86839456-131a5200-c0bf-11ea-9645-50417d8c2218.png)

@amyblais @cpanato 

I tried performing the `bulk import` with the new version but with same configuration earlier. Now I'm getting `Some users were not found` error message. Which in the previous version didn't occur. 

The problem is it's not logging any line number at which this error occurred. Is there anything that could help me debug it ?

**Comment by cpanato** [comment]
@amyblais who is the best person to help for the bulk import?

**Comment by amyblais** [comment]
I believe @grundleborg / @jespino may be able to help with https://github.com/mattermost/mattermost-server/issues/14959#issuecomment-655117485.

**Comment by Shubhamchinda** [comment]
One more thing I'd like to ask. 

I was breaking down the `bulk import` in sections like, first I'll import only the team, then channels and after that with users and messages.

While I was doing performing the `bulk import` with channels and users. I came across this error. 
![Screenshot from 2020-07-08 18-31-42](https://user-images.githubusercontent.com/21038543/86925365-6c7d9200-c14e-11ea-9392-6e30eb999f1b.png)

The error is `CreateChannel: A channel member with that ID already exists`

The `JSONL` data I'm using is `**FROM line 75-84**` looks like:
```
{"type": "channel", "channel": {"team": "demo-team", "name": "z9qyapowejye62yrr", "display_name": "z9qYAPoWEjYe62YrR", "type": "P"}}
{"type": "channel", "channel": {"team": "demo-team", "name": "futures", "display_name": "futures", "type": "O"}}
{"type": "channel", "channel": {"team": "demo-team", "name": "demo-team-lounge", "display_name": "demo-team-lounge", "type": "O"}}
{"type": "channel", "channel": {"team": "demo-team", "name": "tech-support", "display_name": "tech-support", "type": "O"}}
{"type": "channel", "channel": {"team": "demo-team", "name": "4axtzrbi3fttzecxg", "display_name": "4aXtZrbi3FTTzeCXG", "type": "O"}}
{"type": "channel", "channel": {"team": "demo-team", "name": "start-here", "display_name": "start-here", "type": "O"}}
{"type": "channel", "channel": {"team": "demo-team", "name": "scripts-to-trade-support", "display_name": "scripts-to-trade-support", "type": "O"}}
{"type": "user", "user": {"username": "digitalwabbit", "email": "cmnuncio@gmail.com", "roles": "system_user", "teams": [{"name": "demo-team", "channels": [{"name": "s8l4tn2qquenlsb7t", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "bezardev-test-room", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "futures", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "members-trading-floor", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "options-demo-team", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "pro-trading-room", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "scripts-to-trade-support", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "small-accounts", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "small-caps--demo-team", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "start-here", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "stocks-beginner-questions", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "tech-support", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "test1", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "test2", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "test2", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "demo-team-lounge", "notify_props": {"desktop": "default", "mark_unread": "all"}}]}]}}
{"type": "user", "user": {"username": "bezardev2", "password": "OGNEb6c#1W*WJcEUglnJlmgP", "email": "accounts+bezardev2@bezar.io", "roles": "system_admin system_user", "teams": [{"name": "demo-team", "channels": [{"name": "bezardev-test-room", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "futures", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "members-trading-floor", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "pro-trading-room", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "scripts-to-trade-support", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "small-accounts", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "small-caps--demo-team", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "start-here", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "stocks-beginner-questions", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "tech-support", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "demo-team-lounge", "notify_props": {"desktop": "default", "mark_unread": "all"}}]}]}}
{"type": "user", "user": {"username": "demo-teamadmin", "password": "OGNEb6c#1W*WJcEUglnJlmgP", "email": "accounts@demo-team.net", "roles": "system_admin system_user", "teams": [{"name": "demo-team", "channels": [{"name": "4axtzrbi3fttzecxg", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "6nvpwnn9szb5aisbg", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "cjbb6xfsngybateus", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "chyo9fqgc28jmwxxx", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "dfxn2eqcwfcxishs3", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "hy9ddkw6pwddw67d4", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "rsjmrcno8ypca4fie", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "rg77c9xb63zayocfa", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "sdzrkvolg33pmydnz", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "ankdybqjljhbgyxnn", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "bezardev-test-room", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "buddy-system-ideas", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "futures", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "guardian-chat", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "hsanmytmavjbufjc7", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "k7mdnrrvcfpqedanm", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "mastermind", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "members-trading-floor", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "mod-chat", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "options-demo-team", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "pnoyazmkzhbjrlf6f", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "pro-trading-room", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "review-your-trades", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "sdfxds4gs9vf9jpse", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "scripts-to-trade-support", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "small-accounts", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "small-caps--demo-team", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "start-here", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "stocks-beginner-questions", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "tech-support", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "test1", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "demo-team-lounge", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "z9qyapowejye62yrr", "notify_props": {"desktop": "default", "mark_unread": "all"}}]}]}}
```

**Line 82** in error is occurring is : 
```
{"type": "user", "user": {"username": "digitalwabbit", "email": "cmnuncio@gmail.com", "roles": "system_user", "teams": [{"name": "demo-team", "channels": [{"name": "s8l4tn2qquenlsb7t", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "bezardev-test-room", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "futures", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "members-trading-floor", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "options-demo-team", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "pro-trading-room", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "scripts-to-trade-support", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "small-accounts", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "small-caps--demo-team", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "start-here", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "stocks-beginner-questions", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "tech-support", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "test1", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "test2", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "test2", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "demo-team-lounge", "notify_props": {"desktop": "default", "mark_unread": "all"}}]}]}}
```

Basically this is first time that this user is being added with `bulk import` but I don't know why this error is coming up even though there is nothing the MM database.

**Comment by Shubhamchinda** [comment]
> One more thing I'd like to ask.
> 
> I was breaking down the `bulk import` in sections like, first I'll import only the team, then channels and after that with users and messages.
> 
> While I was doing performing the `bulk import` with channels and users. I came across this error.
> ![Screenshot from 2020-07-08 18-31-42](https://user-images.githubusercontent.com/21038543/86925365-6c7d9200-c14e-11ea-9392-6e30eb999f1b.png)
> 
> The error is `CreateChannel: A channel member with that ID already exists`
> 
> The `JSONL` data I'm using is `**FROM line 75-84**` looks like:
> 
> ```
> {"type": "channel", "channel": {"team": "demo-team", "name": "z9qyapowejye62yrr", "display_name": "z9qYAPoWEjYe62YrR", "type": "P"}}
> {"type": "channel", "channel": {"team": "demo-team", "name": "futures", "display_name": "futures", "type": "O"}}
> {"type": "channel", "channel": {"team": "demo-team", "name": "demo-team-lounge", "display_name": "demo-team-lounge", "type": "O"}}
> {"type": "channel", "channel": {"team": "demo-team", "name": "tech-support", "display_name": "tech-support", "type": "O"}}
> {"type": "channel", "channel": {"team": "demo-team", "name": "4axtzrbi3fttzecxg", "display_name": "4aXtZrbi3FTTzeCXG", "type": "O"}}
> {"type": "channel", "channel": {"team": "demo-team", "name": "start-here", "display_name": "start-here", "type": "O"}}
> {"type": "channel", "channel": {"team": "demo-team", "name": "scripts-to-trade-support", "display_name": "scripts-to-trade-support", "type": "O"}}
> {"type": "user", "user": {"username": "digitalwabbit", "email": "cmnuncio@gmail.com", "roles": "system_user", "teams": [{"name": "demo-team", "channels": [{"name": "s8l4tn2qquenlsb7t", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "bezardev-test-room", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "futures", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "members-trading-floor", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "options-demo-team", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "pro-trading-room", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "scripts-to-trade-support", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "small-accounts", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "small-caps--demo-team", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "start-here", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "stocks-beginner-questions", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "tech-support", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "test1", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "test2", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "test2", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "demo-team-lounge", "notify_props": {"desktop": "default", "mark_unread": "all"}}]}]}}
> {"type": "user", "user": {"username": "bezardev2", "password": "OGNEb6c#1W*WJcEUglnJlmgP", "email": "accounts+bezardev2@bezar.io", "roles": "system_admin system_user", "teams": [{"name": "demo-team", "channels": [{"name": "bezardev-test-room", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "futures", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "members-trading-floor", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "pro-trading-room", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "scripts-to-trade-support", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "small-accounts", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "small-caps--demo-team", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "start-here", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "stocks-beginner-questions", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "tech-support", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "demo-team-lounge", "notify_props": {"desktop": "default", "mark_unread": "all"}}]}]}}
> {"type": "user", "user": {"username": "demo-teamadmin", "password": "OGNEb6c#1W*WJcEUglnJlmgP", "email": "accounts@demo-team.net", "roles": "system_admin system_user", "teams": [{"name": "demo-team", "channels": [{"name": "4axtzrbi3fttzecxg", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "6nvpwnn9szb5aisbg", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "cjbb6xfsngybateus", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "chyo9fqgc28jmwxxx", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "dfxn2eqcwfcxishs3", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "hy9ddkw6pwddw67d4", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "rsjmrcno8ypca4fie", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "rg77c9xb63zayocfa", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "sdzrkvolg33pmydnz", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "ankdybqjljhbgyxnn", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "bezardev-test-room", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "buddy-system-ideas", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "futures", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "guardian-chat", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "hsanmytmavjbufjc7", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "k7mdnrrvcfpqedanm", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "mastermind", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "members-trading-floor", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "mod-chat", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "options-demo-team", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "pnoyazmkzhbjrlf6f", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "pro-trading-room", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "review-your-trades", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "sdfxds4gs9vf9jpse", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "scripts-to-trade-support", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "small-accounts", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "small-caps--demo-team", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "start-here", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "stocks-beginner-questions", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "tech-support", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "test1", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "demo-team-lounge", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "z9qyapowejye62yrr", "notify_props": {"desktop": "default", "mark_unread": "all"}}]}]}}
> ```
> 
> **Line 82** in which error is occurring is :
> 
> ```
> {"type": "user", "user": {"username": "digitalwabbit", "email": "cmnuncio@gmail.com", "roles": "system_user", "teams": [{"name": "demo-team", "channels": [{"name": "s8l4tn2qquenlsb7t", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "bezardev-test-room", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "futures", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "members-trading-floor", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "options-demo-team", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "pro-trading-room", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "scripts-to-trade-support", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "small-accounts", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "small-caps--demo-team", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "start-here", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "stocks-beginner-questions", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "tech-support", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "test1", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "test2", "notify_props": {"desktop": "default", "mark_unread": "all"}}, {"name": "demo-team-lounge", "notify_props": {"desktop": "default", "mark_unread": "all"}}]}]}}
> ```
> 
> Basically this is first time that this user is being added with `bulk import` but I don't know why this error is coming up even though there is nothing the MM database.

I solved this one!

**Comment by Shubhamchinda** [comment]
@jespino @grundleborg Could you please help me out here https://github.com/mattermost/mattermost-server/issues/14959#issuecomment-655117485

**Comment by jespino** [comment]
@Shubhamchinda looks like for some reason there is some reference to a user, that is not in the list of users imported at the beginning.

**Comment by Shubhamchinda** [comment]
Yeah, I know... I guess I'd to recheck and verify all the users and users in messages.

**Comment by Shubhamchinda** [comment]
Thanks @jespino I was able to fix the above issue.

Now I was working with attachments.

The JSONL format I passed : 
```
{"type": "post", "post": {"team": "demo", "channel": "futures", "user": "demorobert", "message": "", "props": {}, "create_at": 1584692433594, "attachments": [{"path": "/mm/mongoImages/yfdX9xnNKBeEGEsnT.pdf"}]}}
{"type": "post", "post": {"team": "demo", "channel": "futures", "user": "demorobert", "message": "", "props": {}, "create_at": 1584692604172, "attachments": [{"path": "/mm/mongoImages/dikkY5BaR8wswGnxx.pdf"}]}}
{"type": "post", "post": {"team": "demo", "channel": "demo-lounge", "user": "ajdemo", "message": "", "props": {}, "create_at": 1586022288172, "attachments": [{"path": "/mm/mongoImages/DgKxwAECxumdNyPvq.png"}]}}
```
I got this error : 
![Screenshot from 2020-07-10 06-17-43](https://user-images.githubusercontent.com/21038543/87104526-d6dd2200-c275-11ea-854c-407f226ac38b.png)

Could you please help me here?

**Comment by jespino** [comment]
That error is provided by the MySQL server and is saying that is not possible to write a new post because it doesn't have space for that. Normally it means the partition where the database lives is full. Can you check that?

**Comment by Shubhamchinda** [comment]
Hey @jespino Thanks a lot, that helped!

After solving this I performed the bulk migration with attachments. I got into these error :

> ![Screenshot from 2020-07-17 01-15-19](https://user-images.githubusercontent.com/21038543/87717064-7138ea80-c7cd-11ea-8799-a9a20ae5cf19.png)

First I got these logs `Unable to attach the file`  and then

> ![Screenshot from 2020-07-17 01-22-40](https://user-images.githubusercontent.com/21038543/87717068-726a1780-c7cd-11ea-8477-70b824add26a.png)

I got this error `Unable to save , invalid connection` but I'm not why this occured as the Db server was working fine. 
Could you help?

Thanks :)

**Comment by Shubhamchinda** [comment]
@jespino Did you get the time to look at it?

**Comment by amyblais** [comment]
I'm cleaning up some older issues. If you need further assistance on this, I recommend reaching out to our devs / community via https://forum.mattermost.com/ or via our [Peer-to-peer Help channel](https://community.mattermost.com/core/channels/peer-to-peer-help).
